elasticsearch/index_templates/common.settings.operations.template.json

```diff
@@ -7,5 +7,8 @@
     "index.translog.flush_threshold_size": "256mb",
     "index.unassigned.node_left.delayed_timeout": "2m"
   },
-  "template": ".operations*"
+  "index_patterns": [
+    "infra-*",
+    "audit.infra-*"
```

Comment:
@jcantrill isn't `audit.infra` subsumed by `infra` ?

Reply:
According to https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/cluster-logging-es-rollover-data-design.md#data-model there will be `infra-*` and `audit-*` logs but no `audit.infra-*` which means I need to change it. But I also need to change it in our data model and release a new version of index templates and create another PR.

Reply:
There are currently only 3 patterns we support in index managment:  infra*, app*, audit*